_posts/2019-12-06-website-categories.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: post
+title: Website categories
+date: 2019-12-06 16:28:05.000000000 +01:00
+type: post
+published: true
+status: publish
+categories: [Me]
+image:
+image2:
+author: Valentino Urbano
+---
+
+I'm mostly writing about technology and programming in particular. I plan to continue this trend in the future.
+
+That doesn't mean that my blog is only limited to those topics as the main argument. It represents what I like so it can and will shift over time since I don't like to have different blogs for different areas. It should be the one place where anyone ([me included][1]) can find (almost) everything that I write online.
+
+I try to vary and write about a lot of different topics. I do not want my blog to be e thematic. I want it to be a reflection of who i am and what i find interesting in life.
```

Comment:
E?

Reply:
CAPITALISE I

Reply:
> E?

what?

Reply:
"I do not want my blog to be e thematic." what's e thematic?

Reply:
oh ok